MODULE.bazel

```diff
@@ -1,64 +1,29 @@
-"""remote-apis dependencies"""
-
 module(
-    name = "bazel_remote_apis",
-    repo_name = "com_github_bazelbuild_remote_apis",
+    name = "remoteapis",
+    version = "0",
 )
 
-# deps
 bazel_dep(name = "gazelle", version = "0.35.0")
+bazel_dep(name = "googleapis", version = "0.0.0-20240326-1c8d509c5", repo_name = "com_google_googleapis")
+bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
 bazel_dep(name = "rules_cc", version = "0.0.9")
 bazel_dep(name = "rules_go", version = "0.44.2")
 bazel_dep(name = "rules_java", version = "7.3.2")
 bazel_dep(name = "rules_proto", version = "6.0.0-rc1")
 bazel_dep(name = "rules_proto_grpc_cpp", version = "5.0.0-alpha2")
-bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
-
-# dev deps
-bazel_dep(name = "toolchains_llvm", version = "0.10.3", dev_dependency = True)
-
-# Top-level projects will need a C++ toolchain to build protoc, grpc, etc., but
-# we shouldn't dictate their toolchain, so the toolchain is marked as a dev
-# dependency.
-llvm = use_extension("@toolchains_llvm//toolchain/extensions:llvm.bzl", "llvm", dev_dependency = True)
-llvm.toolchain(
-    # This is a version of LLVM that builds absl and upb successfully with their
-    # default compile options.
-    #
-    # TODO: Update this to the latest version of LLVM possible
-    llvm_version = "13.0.0",
-)
-use_repo(llvm, "llvm_toolchain")
-register_toolchains("@llvm_toolchain//:all", dev_dependency = True)
```

Comment:
this version doesn't support macOS, excluding it just replies on the host having a CC toolchain installed, we could probably bring this back with a different version to support macOS too but I think the norm is normally to expect the host to have something for this